src/components/Form/Form.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import { ReactElement, useState } from 'react'
+import { Flex } from 'antd'
+import { Meta } from '@storybook/react'
+
+import * as validators from './utils/validators.ts'
+import { Button } from '../Button'
+import { Card } from '../Card'
+import { Checkbox } from '../Checkbox'
+import { CheckboxGroup } from '../CheckboxGroup'
+import { Form } from './Form.tsx'
+import { FormItem } from './FormItem.tsx'
+import { Input } from '../Input'
+import { InputNumber } from '../InputNumber'
+import { RadioGroup } from '../RadioGroup'
+import { Search } from '../Search'
+import { Select } from '../Select'
+import { Switch } from '../Switch'
+import { TextArea } from '../TextArea'
+
+const meta = {
+  component: Form,
+  args: {},
+  argTypes: {},
+} satisfies Meta<typeof Form>
+
+export default meta
+// type Story = StoryObj<typeof meta>
+
+const options = [
+  ...Array(3).keys(),
+].map((id) => ({
+  value: `value ${id + 1}`,
+  label: `label ${id + 1}`,
+}))
+
+const formValuesDebugger = (values: Record<string, unknown>): ReactElement => {
+  return (
+    <div style={{ gridColumn: '-1 / 1', marginBottom: 24 }}>
+      <Card>
+        <pre>{JSON.stringify(values, null, '\t')}</pre>
+      </Card>
+    </div>
+  )
+}
+
+export const ComplexForm = (): ReactElement => {
```

Comment:
maybe a few stories about the submit button possibilities are useful for the final user

Reply:
added few stories in here https://github.com/mia-platform/design-system/blob/IDP-94/form-analysis/src/components/Form/Form.stories.tsx#L133 and here for every form item https://github.com/mia-platform/design-system/blob/IDP-94/form-analysis/src/components/Form/FormItem/FormItem.stories.tsx